integration/int_3Ri_2BhaRi2_3Re_2BhaRe3/brokers/tasks/copy_config_docker_host_edg.yml

```diff
@@ -17,3 +17,6 @@
   synchronize:
     src: "{{ role_path }}/files/broker-edge/slave/"
     dest: "{{ role_path }}/files/broker-edge/slave"
+
+- name: "Changing SELinux context to allow containers to mount volumes"
+  command: "chcon -Rt svirt_sandbox_file_t {{ role_path }}/files/broker-edge/"
```

Comment:
maybe to avoid duplication 
```
tasks:
- import_tasks: tasks/change_selinux_ctx.yml
  vars:
    router_mode:edge/interior
```
and then using something like:
```
- name: "Changing SELinux context to allow containers to mount volumes"
  command: "chcon -Rt svirt_sandbox_file_t {{ role_path }}/files/broker-{{router_mode}}/"
```

Reply:
Maybe it wont be needed. Look at the PR proposed by @VeciCZ.
I think his suggestion is right and just mounting volumes with ":Z" might eliminate the need for changing the selinux context.